Source/GSXib5KeyedUnarchiver.m

```diff
@@ -0,0 +1,3391 @@
+        }
+      else if (!sel)
+        {
+          NSWarnMLog(@"label %@ does not correspond to any selector", label);
+        }
+      else if (!trigsel)
+        {
+          NSWarnMLog(@"trigger %@ does not correspond to any selector", trigger);
+        }
+      else
+        {
+          NSWarnMLog(@"destination class (%@) does not respond to trigger selector %@",
+                     NSStringFromClass([destination class]), selName);
+        }
+
+      // PREMATURE RETURN...
+      return;
+    }
+
+  // Otherwise invoke the super class' method...
+  [super establishConnection];
+}
+
+@end
+
+@interface IBOutletConnection5 : IBOutletConnection
+@end
+
+@implementation IBOutletConnection5
+
+- (instancetype) initWithCoder: (NSCoder *)coder
+{
+  self = [super initWithCoder: coder];
+  if (self)
+    {
+      if ([coder allowsKeyedCoding])
+        {
+          // label string tag has changed for XIB5...
+          ASSIGN(label, [coder decodeObjectForKey: @"property"]);
+          // destination  and source string tags are still the same (so far) and loaded
+          // by base class...
+        }
+      else
+        {
+          [NSException raise: NSInvalidArgumentException
+                      format: @"Can't decode %@ with %@.",NSStringFromClass([self class]),
+           NSStringFromClass([coder class])];
+        }
+    }
+  return self;
+}
+
+@end
+
+@interface IBUserDefinedRuntimeAttribute5 : IBUserDefinedRuntimeAttribute
+@end
+
+
+@implementation IBUserDefinedRuntimeAttribute5
+
+- (id) initWithCoder: (NSCoder *)coder
+{
+  self = [super initWithCoder: coder];
+
+  if (self)
+    {
+      if ([coder allowsKeyedCoding])
+        {
+          [self setTypeIdentifier: [coder decodeObjectForKey: @"type"]];
+
+          // Decode value properly...
+          if ([@"boolean" isEqualToString: typeIdentifier])
+            [self setValue: [NSNumber numberWithBool: ([@"YES" isEqualToString: value] ? YES : NO)]];
+          else if ([@"image" isEqualToString: typeIdentifier])
+            [self setValue: [NSImage imageNamed: value]];
+          else if ([@"number" isEqualToString: typeIdentifier])
+            [self setValue: [coder decodeObjectForKey: @"value"]];
+          else if ([@"point" isEqualToString: typeIdentifier])
+            [self setValue: [coder decodeObjectForKey: @"value"]];
+          else if ([@"size" isEqualToString: typeIdentifier])
+            [self setValue: [coder decodeObjectForKey: @"size"]];
+          else if ([@"rect" isEqualToString: typeIdentifier])
+            [self setValue: [coder decodeObjectForKey: @"value"]];
+          else if ([@"nil" isEqualToString: typeIdentifier])
+            [self setValue: nil];
+          else
+            NSWarnMLog(@"type: %@ value: %@ (%@)", typeIdentifier, value, [value class]);
+        }
+    }
+
+  return self;
+}
+
+@end
+
+
+@implementation GSXib5KeyedUnarchiver
+
+static NSDictionary *XmltagToObjectClassCrossReference = nil;
```

Comment:
Also, maybe `s/CrossReference/Mapping` or `s/CrossReference//` or `s/CrossReference/Map` (the last one would match how the suffix is used in name of `XmlTagToDecoderSelectorMap`).